src/main/java/net/minestom/server/entity/type/projectile/EntityProjectile.java

```diff
@@ -31,19 +31,22 @@
     public EntityProjectile(@Nullable Entity shooter, @NotNull EntityType entityType) {
         super(entityType);
         this.shooter = shooter;
+        setup();
     }
 
     @Deprecated
     public EntityProjectile(@Nullable Entity shooter, @NotNull EntityType entityType, @NotNull Position spawnPosition) {
         super(entityType, spawnPosition);
         this.shooter = shooter;
+        setup();
     }
 
     private void setup() {
         super.hasPhysics = false;
         if (getEntityMeta() instanceof ProjectileMeta) {
             ((ProjectileMeta) getEntityMeta()).setShooter(this.shooter);
         }
+        setGravity(0.02f, 0.04f, 1.96f);
         setBoundingBox(.5F, .5F, .5F);
```

Comment:
I didn't check this previously, but any reason why the bounding box is hardcoded instead of using `EntityType`?